node_filesystem/templates/felix.cfg.template

```diff
@@ -1,5 +1,6 @@
 [global]
 MetadataAddr  = None
+IpInIpEnabled = True
```

Comment:
Agree, but leaving as is for MVP.  Will track this getting done at a later date.